nc/include/component/Collider.h

```diff
@@ -11,15 +11,24 @@ namespace nc
         public:
             Collider(EntityHandle handle, Vector3 scale);
             ~Collider();
-            void Update(graphics::FrameManager& frame);
+
+            void UpdateTransformationMatrix();
+            DirectX::FXMMATRIX GetTransformationMatrix() const;
 
             #ifdef NC_EDITOR_ENABLED
+            void UpdateModel(graphics::FrameManager& frame);
```

Comment:
"Model" is a bit confusing as a name here, even though that's what it is. When I see this I think the collider somehow has control over the model it's attached to (Bee, Sphere, etc.) and its not apparent this is referring to the wireframe collision widget.